corehq/apps/api/tests/user_resources.py

```diff
@@ -277,7 +277,7 @@ def test_update_profile_conflict(self):
 
 class TestWebUserResource(APIResourceTest):
     """
-    Basic sanity checking of v0_1.CommCareUserResource
+    Basic sanity checking of v0_5.WebUserResource
     """
     resource = v0_5.WebUserResource
     api_name = 'v0.5'
```

Comment:
looks like this means that we don't have tests for v0_1.

Reply:
Yeah that's what it looks like